### DISCUSSION TITLE
Image upload is broken on the mobile app

### DISCUSSION THREAD

**Discussion by andreynering** [community/community#43595] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I was writing #43594 and image upload simply wasn't working. I tried multiple times, but in the end I had to publish without the image and edit it on the browser to add it.

![Screenshot_20230107-182530_GitHub](https://user-images.githubusercontent.com/7011819/211171011-cf483bc7-d357-4bd3-93f8-4417620e9536.jpg)

**Comment by stevepopovich** [comment, marked as answer]
Hi @andreynering! We are aware of this issue and have a fix coming out for it in the next few releases. Thanks for the report!
